1-exercises/C-undefined-properties/exercise.js

```diff
@@ -25,10 +25,10 @@ let user = {
   name: "Mira"
 };
 
-sayHelloToUser(user);
+sayHelloToUser(user); // there is no first name property in the user function. the console target the user function causing an undefined to appear
 
 // Example 3
-let myPet = {
+let myPet = { // getName" is not a function, "getName" is a property with a value us a function in myPet Object not a function.
```

Comment:
Undefined here is related to function. Try to add a return to function like this:

`  getName: function () {
    return "My pet's name is Fluffy";
  },`
  
  Please check this: https://stackoverflow.com/questions/17337064/does-every-javascript-function-have-to-return-a-value